ref/src/print_affinity.c

```diff
@@ -6,10 +6,10 @@
 #include <omp.h>
 #endif
 
-#ifndef __APPLE__
 extern void runnable (cpu_set_t *, int *, int *);
 
-void print_affinity_ (int *rank)
+#ifndef __APPLE__
+void print_affinity_ ()
```

Comment:
@LyndStringer - This fails the MacOS build check because you moved the `#ifndef __APPLE__` so that it does not protect the declaration of the `runnable` external function.  You can't do that because `cpu_set_t` is not defined on Mac.